contracts/lib/Errors.sol

```diff
@@ -255,6 +255,9 @@ library Errors {
     /// @notice Zero address provided for IP Graph ACL.
     error LicenseRegistry__ZeroIPGraphACL();
 
+    /// @notice license terms disabled.
+    error LicenseRegistry__LicenseTermsDisabled(address ipId, address licenseTemplate, uint256 licenseTermsId);
```

Comment:
It seems this error isn't used anywhere in the code. Could it be redundant? Just wanted to confirm.